exotic.py

```diff
@@ -371,111 +376,49 @@ def user_input(prompt, type_, val1=None, val2=None):
 # Create a save directory within the current working directory
 def create_directory():
     try:
-        directoryName = input('Enter the name for your new directory: ')
-        newDirectoryPath = os.getcwd()
-        saveDirectory = newDirectoryPath + '/' + directoryName + '/'
-        os.mkdir(saveDirectory)
+        directoryname = input('Enter the name for your new directory: ')
+        newdirectoryPath = os.getcwd()
+        savedirectory = newdirectoryPath + '/' + directoryname + '/'
+        os.mkdir(savedirectory)
     except OSError:
-        print("Creation of the directory %s failed" % saveDirectory)
+        print("Creation of the directory %s failed" % savedirectory)
```

Comment:
If the code is unable to create the directory, then this function will fail and the directory will not be created, potentially crashing EXOTIC. Maybe you should use a `while` loop here like you've done with other checks?

Reply:
Done!